monai/fl/client/monai_algo.py

```diff
@@ -328,28 +298,23 @@ class MonaiAlgo(ClientAlgo, MonaiAlgoStats):
     FIXME: reimplement this class based on the bundle "ConfigWorkflow".
```

Comment:
Should this line be removed now?

Reply:
Oh, yes, will remove it soon.

Thanks.

Reply:
Removed in the latest commit.

Thanks.